client/src/pages/Dashboard/components/AllSurveysTable.module.css

```diff
@@ -0,0 +1,27 @@
+.row {
+  background: white;
+  transition: background 0.3s;
+  cursor: pointer;
+  text-decoration: none;
+}
+
+.row:hover {
+  background: #b4dbff7a;
+}
+
+.table {
+  box-shadow: 0px 2px 1px -1px rgba(0, 0, 0, 0.2),
```

Comment:
is this the Material UI box shadow or have you written your own? If you have then we need to make sure it's consistent across components

Reply:
done